src/server/routes/units.js

```diff
@@ -41,16 +43,23 @@ router.post('/edit', async (req, res) => {
 	const validUnit = {
 		type: 'object',
 		required: ['id', 'identifier'],
```

Comment:
You didn't do this but it should be fixed/tested. The code below seems to change all the unit values to the ones sent even if they do not exist. This would seem to change a unit incorrectly if those values are not supplied. I think they all should be required. One could default on some but I think it best to require all the values. Let me know if you need help in modifying the code.

Reply:
I created the issue.